google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/SpannerServiceContributor.java

```diff
@@ -32,10 +37,27 @@
  */
 public class SpannerServiceContributor implements ServiceContributor {
 
+  private static final SpannerSchemaManagementTool SCHEMA_MANAGEMENT_TOOL =
+      new SpannerSchemaManagementTool();
+
   static final String HIBERNATE_API_CLIENT_LIB_TOKEN = "sp-hib";
 
+  @Override
   public void contribute(StandardServiceRegistryBuilder serviceRegistryBuilder) {
     serviceRegistryBuilder
-        .applySetting("hibernate.connection.userAgent", HIBERNATE_API_CLIENT_LIB_TOKEN);
+        .applySetting("hibernate.connection.userAgent", HIBERNATE_API_CLIENT_LIB_TOKEN)
+        .addInitiator(
```

Comment:
Try also just setting the hibernate property directly.

Reply:
Done. This works!